internal/sdkprovider/service/organization/organization_user_group_test.go

```diff
@@ -128,3 +131,71 @@ func testAccCheckAivenOrganizationUserGroupAttributes(n string) resource.TestChe
 		return nil
 	}
 }
+
+func TestAccAivenOrganizationUserGroup_Import(t *testing.T) {
+	var (
+		resourceName = "aiven_organization_user_group.import_group"
+		rName        = acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+		orgID        string
+		groupID      string
+	)
+
+	orgID = os.Getenv("AIVEN_ORG_ID")
+	if orgID == "" {
+		t.Skip("Skipping test due to missing AIVEN_ORG_ID or AIVEN_ORG_USER_EMAIL environment variable")
```

Comment:
AIVEN_ORG_USER_EMAIL is not mention anywhere in the test. Is it required for real?